src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportFileWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements ReportFileWriter {
+
+    @Override
+    public void write(String report, String pathToFile) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(pathToFile))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can not write to file " + pathToFile);
```

Comment:
When rethrowing the exception, it's recommended to include the original exception object to preserve the stack trace. This makes debugging easier if an error occurs. Consider changing the catch block to include the original IOException, e.g., `throw new RuntimeException("Can not write to file " + pathToFile, e);`.